.github/workflows/CI.yml

```
@@ -0,0 +1,43 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.3'
+          - '1'
```

```suggestion
          - 'lts'
          - '1'
          - 'pre'
```

assuming a bump to LTS